homepage/templates/homepage/app_layout.html

```diff
@@ -48,6 +54,24 @@
   </div>  
 </nav>
 
+
+<style>
+  .messages {
+    background-color:dodgerblue;
+    color: white;
+    padding: 10px;
+  }
+  </style>

```

Comment:
<style> should live in between the <head> tags

Reply:
Added <head> tags

Reply:
@LeonRabinovich , `<head>` tags only appear once per html page, above the `<body>` tag.
See [here](https://www.w3schools.com/html/html_intro.asp) for further reference

Reply:
@VeraTs 
Fixed, thank you for the information.